addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/tags/PolicyTags.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.commonlib.tags;
+
+public enum PolicyTags {
+
+    /**
+     * Tags to be associated with standardized scan policies which will be distributed as an add-on.
+     *
+     * @since 1.29.0
+     */
+    POLICY_DEV_CICD,
+    POLICY_DEV_STD,
+    POLICY_DEV_FULL,
+    POLICY_QA_STD,
+    POLICY_QA_FULL,
+    POLICY_SEQUENCE,
+    POLICY_API;
```

Comment:
`POLICY_CASA`?

Reply:
Uh, I hadn't planned that for this sprint.

I guess I can add it here anyway and tackle it in the other pieces/places if time permits.